modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -71,6 +71,14 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also includes the following bug fixes:
 
+=== Backspace would in some situations delete the image after the cursor instead of before it as expected.
+//#TINY-9807
+
+In previos versions of {productname}, after pressing backspace occasionally would deleted a image after the cursor instead of the expected deletion before it.
```

Comment:
```suggestion
In previous versions of {productname}, after pressing backspace occasionally would delete an image after the cursor instead of the expected deletion before it.
```